src/firebase/api.jsx

```diff
@@ -0,0 +1,24 @@
+import { getDatabase } from "firebase/database";
+import { initializeApp } from "firebase/app";
+import { getAnalytics } from "firebase/analytics";
+
+
+
+const firebaseConfig = {
+    apiKey: "AIzaSyCo0gPK4fEqqPd5WgmhISKfy1P8AgPmf3Y",
```

Comment:
i don't know if we should expose the api key like this

Reply:
We should probably omit this file/comment the key out